src/VoteDelegate.sol

```diff
@@ -95,6 +104,11 @@ contract VoteDelegate {
         emit Free(msg.sender, wad);
     }
 
+    function reserveHatch() external {
+        require(block.number > hatchTrigger + HATCH_SIZE + HATCH_COOLDOWN, "VoteDelegate/cooldown-not-finished");
+        hatchTrigger = block.number;
```

Comment:
Wondering if it could be worth it to add an event here to make it easier for a UI to help lockers target the cooldown window. On the other hand the DoS edge case may not be worth it for any UI to care about so not sure. Though I guess if you are fighting a targeted DOS attack it may help being able to easily fetch all recent reserveHatch() to plan your move.

Reply:
Done.